jerry-core/parser/js/opcodes-dumper.cpp

```diff
@@ -352,6 +352,16 @@ dumper_try_replace_identifier_name_with_reg (scopes_tree tree, /**< a function s
         break;
       }
 
+      /*
```

Comment:
Currently, comment at line 346 is not correct.
Could you, please, update it to the following?
```
       /*
       * 'assignment' and 'meta' are the only opcodes with statically unspecified argument type
       * (checked by assertions above)
       */
```